Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -291,22 +292,25 @@ def getJobStatus(self, jobIDList):
       job, jobID = condorIDFromJobRef(jobRef)
       condorIDs[job] = jobID
 
-    # This will return a list of 1245.75 3
-    status, stdout_q = commands.getstatusoutput(
+    qList = []
+    for _condorIDs in breakListIntoChunks(condorIDs.values(), 100):
+      
+      # This will return a list of 1245.75 3
+      status, stdout_q = commands.getstatusoutput(
         'condor_q %s %s -af:j JobStatus ' %
-        (self.remoteScheddOptions, ' '.join(
-            condorIDs.values())))
-    if status != 0:
-      return S_ERROR(stdout_q)
-    qList = stdout_q.strip().split('\n')
+        (self.remoteScheddOptions, ' '.join(_condorIDs)))
+      if status != 0:
+        return S_ERROR(stdout_q)
+      _qList = stdout_q.strip().split('\n')
+      qList.extend(_qList)
 
-    # FIXME: condor_history does only support j for autoformat from 8.5.3,
-    # format adds whitespace for each field This will return a list of 1245 75 3
-    # needs to cocatenate the first two with a dot
-    condorHistCall = 'condor_history %s %s -af ClusterId ProcId JobStatus' % (
-        self.remoteScheddOptions, ' '.join(condorIDs.itervalues()))
+      # FIXME: condor_history does only support j for autoformat from 8.5.3,
+      # format adds whitespace for each field This will return a list of 1245 75 3
+      # needs to cocatenate the first two with a dot
+      condorHistCall = 'condor_history %s %s -af ClusterId ProcId JobStatus' % (
+        self.remoteScheddOptions, ' '.join(_condorIDs))
 
-    treatCondorHistory(condorHistCall, qList)
+      treatCondorHistory(condorHistCall, _qList)
```

Comment:
`treatCondorHistory` appends entries to qList, so you have to use the list that is kept
```suggestion
      treatCondorHistory(condorHistCall, qList)
```

Reply:
Ah, I see... I haven't realized it.
Thanks a lot!